packages/next/src/templates/Default/index.tsx

```diff
@@ -51,6 +51,15 @@ export const DefaultTemplate: React.FC<DefaultTemplateProps> = ({
   const { Actions } = React.useMemo<{
     Actions: Record<string, React.ReactNode>
   }>(() => {
+    const viewActionServerProps = {
```

Comment:
Rename this to `serverProps` and type is as `ServerProps` (imported from `payload`). Can these also be defined outside the `useMemo`?

Reply:
They can but they still have to be memoized. I can also pass these "shared" server props to the other components defined in the same file.

Reply:
If that's an issue we can memoize the `serverProps` object itself, and then pass that around to the other components in this layout like you've suggested.

Reply:
@jacobsfletch I also realized that the `avatar` and `graphics.Icon` here are not recieving serverProps - shall I include them?

Reply:
Moved and memoized serverProps, added type as discussed.